pkg/engine/datasource/introspection_datasource/planner.go

```diff
@@ -0,0 +1,56 @@
+package introspection_datasource
+
+import (
+	"encoding/json"
+
+	"github.com/jensneuse/graphql-go-tools/pkg/engine/plan"
+	"github.com/jensneuse/graphql-go-tools/pkg/introspection"
+)
+
+type Planner struct {
+	introspectionData *introspection.Data
+	v                 *plan.Visitor
```

Comment:
I think I would have guessed `visitor` with `v` ... but why not make it really clear and write it out? :-)